src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/IProjectTreeExtensions.cs

```diff
@@ -6,7 +6,6 @@ namespace Microsoft.VisualStudio.ProjectSystem.VS
 {
     internal static class IProjectTreeExtensions
     {
-        public static HierarchyId GetHierarchyId(this IProjectTree tree) =>
-            new HierarchyId(tree.IsRoot() ? VSConstants.VSITEMID_ROOT : unchecked((uint)tree.Identity));
+
```

Comment:
i moved it to the root and namespace Microsoft.VisualStudio.ProjectSystem.VS to have only one extensions file

---
In reply to: [74471919](https://github.com/dotnet/roslyn-project-system/pull/349#discussion_r74471919) [](ancestors = 74471919)